bench_manager/bench_manager/doctype/bench_settings/bench_settings.py

```diff
@@ -383,3 +383,31 @@ def run_command(commands, doctype, key, cwd="..", docname=" ", after_command=Non
 			docname=docname,
 			commands=commands,
 		)
+
+def backup_daily_sites():
+    site_list = frappe.get_list("Site",filters={"frequancy":"Daily","auto_backup":1})
+    if site_list:
+        create_backup(site_list)
+def backup_weekly_sites():
+    site_list = frappe.get_list("Site",filters={"frequancy":"Weekly","auto_backup":1})
+    if site_list:
+        create_backup(site_list)
+
+def backup_monthly_sites():
+    site_list = frappe.get_list("Site",filters={"frequancy":"Monthly","auto_backup":1})
+    if site_list:
+        create_backup(site_list)
+
+def create_backup(site_list):
+    key = datetime.now()
+    for i in site_list:
+        site_doc = frappe.get_doc("Site",i.name)
+        key += timedelta(seconds=1)
+        frappe.enqueue(
```

Comment:
This will be a massive I/O operation and it will kill the server I prefer to do that without enqueueing and put try/catch to be sure the process continues even if there is an issue with one site
Please add alerts for this if any issue happened